spec.bs

```diff
@@ -5398,7 +5401,7 @@ To <dfn>look up per-buyer multi-bid limit</dfn> given an [=auction config=] |auc
 <h3 id=bid-generators>Bid generator</h3>
 
 A <dfn>per buyer bid generator</dfn> is an [=ordered map=] whose [=map/keys=] are [=URLs=]
-representing [=interest group/trusted bidding signals urls=], and whose [=map/values=] are
+representing [=interest group/trusted bidding signals urls=] or null, and whose [=map/values=] are

```

Comment:
This might be more clear as:

```
A <dfn>per buyer bid generator</dfn> is an [=ordered map=] whose [=map/keys=] are [=URLs=] or null
representing [=interest group/trusted bidding signals urls=],
```

(basically, move the `or null` around for readability).